### PR TITLE
Fix: override template issue on templates dir

### PR DESCRIPTION
**Pull request by tokitouq**
Fix template overriding due to wrong package templates structure.
Relating: #3

### CODE REVIEW

File: examples/djangoproject/settings.py [hunk 1]
```diff
@@ -18,6 +19,7 @@
     'django.contrib.staticfiles',
     # 3rd party
     'rest_framework',
+    # package should be at bottom of your apps
```
Comment (baseplate-admin) [inline examples/djangoproject/settings.py:22]: ordering doesn't matter

File: examples/djangoproject/settings.py [hunk 2]
```diff
@@ -39,7 +39,8 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        # configure root templates
+        'DIRS': [os.path.join(BASE_DIR, "templates")],
```
Comment (baseplate-admin) [inline examples/djangoproject/settings.py:43]: this is also unnecessary

File: README.md [hunk 3]
```diff
@@ -29,7 +29,25 @@ REST_FRAMEWORK = {
     )
 }
 ```
-All set! now run app and visit your API endpoint.\
+
+## Override templates
+To override providing templates:
+1. Configure root `templates` dir in your django app
+```bash
```
Comment (baseplate-admin) [inline README.md:36]:
```python

File: examples/djangoproject/settings.py (hunk 1 above)
Comment (tokitouq) [reply examples/djangoproject/settings.py:22]: No it does :imp:

File: examples/djangoproject/settings.py (hunk 2 above)
Comment (tokitouq) [reply examples/djangoproject/settings.py:43]: This is necessary :imp: 
Unlike you I test every case...

Comment (baseplate-admin) [reply examples/djangoproject/settings.py:43]: > This is necessary 👿

Actually no. Check how django registers app. This is actually not necessary

File: examples/djangoproject/settings.py (hunk 1 above)
Comment (baseplate-admin) [reply examples/djangoproject/settings.py:22]: It doesn't lol

**Pull request merged**